components/feature/downloads/src/test/java/mozilla/components/feature/downloads/AbstractFetchDownloadServiceTest.kt

```diff
@@ -778,7 +782,65 @@ class AbstractFetchDownloadServiceTest {
 
         service.setDownloadJobStatus(downloadJobState, COMPLETED)
         assertEquals(COMPLETED, service.getDownloadJobStatus(downloadJobState))
-        testDispatcher.advanceTimeBy(750)
+        testDispatcher.advanceTimeBy(PROGRESS_UPDATE_INTERVAL)
         assertEquals(1, shadowNotificationService.size())
     }
+
+    @Test
+    fun `how many seconds have passed since the last update to a notification`() = runBlocking {
```

Comment:
nit: maybe better `keeps track of how many...` or `measures how many...`

Reply:
👍